SDAPackage/SDA/stationary_obstacle.py

```diff
@@ -31,3 +31,22 @@ def get_height(self):
         The height of the obstacle
         """
         return self.height
+
+    def make_avoidance_points(self, drone_altitude):
+        """
+
```

Comment:
You should either remove this line or add a docstring text. I recommend removing this newline

Reply:
done